src/sentry/models/artifactbundle.py

```diff
@@ -161,6 +161,7 @@ class ArtifactBundleFlatFileIndex(Model):
     project_id = BoundedBigIntegerField(db_index=True)
     release_name = models.CharField(max_length=250)
     dist_name = models.CharField(max_length=64, default=NULL_STRING)
+    size = BoundedPositiveIntegerField(default=0)
```

Comment:
Should this be a BIGINT? Or are customer file sizes always going to fit in 32bits?

Reply:
Yes they would. But I’m not going to move forward with this PR as is anyway. Thanks for the feedback though.